src/services/user/registration.js

```diff
@@ -199,8 +199,8 @@ const registerUser = function(data, params, app) {
 		//store consent
 		consent = {
 			form: 'digital',
-			privacyConsent: data.privacyConsent,
-			termsOfUseConsent: data.termsOfUseConsent,
+			privacyConsent: (data.privacyConsent || data.parent_privacyConsent) === 'true',
```

Comment:
it seems that parent logic should not affect the user consent

Reply:
Switched to a better readable solution